<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWindow</class>
 <widget class="QWidget" name="LoggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logs</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../../data/resources/mainwindow.qrc">
    <normaloff>:/images/mask-icon.png</normaloff>:/images/mask-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnDebug">
       <property name="text">
        <string>Debug</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/script-error.png</normaloff>:/images/oxygen-icons/script-error.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInfo">
       <property name="text">
        <string>Info</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-information.png</normaloff>:/images/oxygen-icons/dialog-information.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnWarning">
       <property name="text">
        <string>Warning</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-warning.png</normaloff>:/images/oxygen-icons/dialog-warning.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnError">
       <property name="text">
        <string>Error</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/dialog-error.png</normaloff>:/images/oxygen-icons/dialog-error.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCritical">
       <property name="text">
        <string>Critical</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/edit-bomb.png</normaloff>:/images/oxygen-icons/edit-bomb.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Save to file</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../data/resources/loggerwindow.qrc">
         <normaloff>:/images/oxygen-icons/document-save-as.png</normaloff>:/images/oxygen-icons/document-save-as.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextBrowser" name="txtLogHistory"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Filter by:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leFilterBy"/>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="cbCaseInsensitive">
     <property name="text">
      <string>Case Insensitive</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPreviousPage">
       <property name="text">
        <string>&lt;&lt; &amp;Previous page</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblLogsStatus">
       <property name="text">
        <string>Showing N of M logs. Page X of Y.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnNextPage">
       <property name="text">
        <string>&amp;Next page &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btnDebug</tabstop>
  <tabstop>btnInfo</tabstop>
  <tabstop>btnWarning</tabstop>
  <tabstop>btnError</tabstop>
  <tabstop>btnCritical</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>txtLogHistory</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../data/resources/loggerwindow.qrc"/>
  <include location="../../../../../data/resources/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
